numpy/core/_add_newdocs.py

```diff
@@ -5491,6 +5491,39 @@
 
     """))
 
+add_newdoc('numpy.core.multiarray', 'dtype', ('metadata',
+    """
+    Either ``None`` or a readonly dictionary of metadata (mappingproxy).
+
+    The metadata field can be set using any dictionary at data-type
+    creation. Note that whether or not operations on arrays with metadata
+    attached to their datatypes is currently not well defined and should
+    not be relied on.
```

Comment:
This sentence seems to be missing a word or two. Was the intent to say:
```suggestion
    creation. Note that whether operations on arrays with metadata
    attached to their datatypes preserve that metadata is currently
    not well defined and should not be relied on.
```

Reply:
I commit the suggestion after changing "preserve" to "propagate", which indeed makes more sense, since the old one is never lost.